### ISSUE TITLE
Object List Item: ‘Selection’ message strip (green colored) is not included in the tab chain. Due to which information about the selected list item is not conveyed.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3837]
Description: ‘Selection’ message strip (green colored) is not included in the tab chain. Due to which information about the selected list item is not conveyed.

Expected: Selection information should be conveyed on selecting the selectable list items within the application.

Screenshot: 

![image](https://user-images.githubusercontent.com/32538291/98775081-60625380-2412-11eb-97fa-1cff187321fd.png)
